packages/contracts/contracts/L1/ZKProofVerifier.sol

```diff
@@ -200,11 +200,11 @@ contract ZKProofVerifier is ISemver {
         _validatePublicInputOutput(
             _storedSrcOutput,
             _storedDstOutput,
-            bytes32(_zkVmProof.publicValues[:32]),
-            bytes32(_zkVmProof.publicValues[32:64])
+            bytes32(_zkVmProof.publicValues[8:40]),
```

Comment:
How about removing the prefix in challenger client?

Reply:
Check the length of `publicValues`, if it is longer than 96, then check if each value has the specific prefix and remove the prefix? Also I think it's better to add a require phrase in verifyZkVmProof contract function, wdyt?

Reply:
Or it's good to check it at the front of proveFaultWithZkVm function to save gas

Reply:
I've tested it with the public values without the prefix, but the verification fails in the SP1Verifier side. I think they need the prefix for the successful verification.

Reply:
Oh really? I understand it, thanks!

Reply:
Then could you add some comments here?